draft-voit-rats-attestation-results.md

```diff
@@ -196,10 +204,10 @@ For the Attesting Environment identity, there MUST exist a chain of trust ultima
 It is upon this root of trust that unique, non-repudiable identities may be founded.
 Example attested identities may include:
 
-* a type of hardware chip used for the Attesting Environment
-* a specific instance of a running Attesting Environment
-* a software build executing within an Attesting Environment
-* the developer(s) responsible for the code executing within an Attesting Environment
+* a type of hardware chip used for the Attesting Environment (e.g., TPM2.0)
+* a unique instance of a running Attesting Environment (e.g., LDevID {{802.1AR}}, Implementation ID {{-PSA}})
```

Comment:
```suggestion
* a unique instance of a running Attesting Environment (e.g., LDevID {{802.1AR}}, Instance ID {{-PSA}})
```

Reply:
I initially placed Instance ID there.  But further reading led me to believe that multiple Instance ID's could be loaded into one TrustZone.  Could you confirm it is 1:1 between Attesting Environment and Instance ID?    I actually expect we are going to need to build more explicit mappings of identities in this section (or the Appendices) over time.

Reply:
Implementation ID is a kind of "class identifier" that is common to all devices sharing the same PSA RoT implementation.  Instance ID is the unique identifier of a PSA RoT instance.